include/swift/AST/DiagnosticsSema.def

```diff
@@ -2952,7 +2952,7 @@ ERROR(invalid_nonobjc_decl,none,
 ERROR(objc_in_extension_context,none,
       "members of constrained extensions cannot be declared @objc", ())
 ERROR(objc_in_generic_extension,none,
-      "@objc is not supported within extensions of generic classes", ())
+      "@objc is not supported within extensions of generic classes, or classes that inherit from generic classes", ())
```

Comment:
I hate to be this nitpicky but I don't think we should use the comma. The diagnostic is one thought.